types/twitch-ext/index.d.ts

```diff
@@ -170,6 +176,44 @@ interface TwitchExtActions {
 	requestIdShare(): void;
 }
 
+/**
+ * @see TwitchExt.configuration
+ */
+interface TwitchExtConfiguration {
+	/**
+	 * This property returns the record for the broadcaster segment if one is found; otherwise, undefined.
+	 */
+	broadcaster: { version: string; content: string } | undefined;
```

Comment:
```suggestion
	broadcaster?: { version: string; content: string };
```
The `?` syntax is more natural here.